### PR TITLE
Add icons to the projects

### PR DESCRIPTION
**Pull request by chrysn**
indicating what they contain.

The PNGs are generated rather than hotlinked also to avoid data leakage, but most importantly because GitHub markdown doesn't have an option to give an image size for reused images.

### CODE REVIEW

**Comment by chrysn** [comment]
@maribu, can you verify the properties? AIU your repos are under an open hardware license, while the miniloon has hardware provided but not under an OSHW license, might add an extra icon (or ask Lasse to clarify).

File: README.md [hunk 1]
```diff
@@ -23,11 +23,21 @@ As part of the RIOT project, the [RIOT code of conduct] applies.
...
-* [Marian's Business Card](https://github.com/maribu/business-card): Hardware description, board support, games and text display software for business card sized board with USB power supply.
-* [Marian's demo boards](https://github.com/maribu/boards): Various PCBs, including an RGB Christmas tree board, along with RIOT board support for most boards.
+Icons indicate that the project offers
+hardware descriptions (![oshw]),
+a RIOT board description (![board])
+or RIOT applications (![app]).
+<!-- Right now we have nothing on the list that is modules only, could get https://game-icons.net/1x1/lorc/jigsaw-piece.html as an icon then -->
+
+* ![board] ![app] [MiniLoon](https://www.hackster.io/386121/miniloon-60a9c9): A RIOT powered balloon with sensors to monitor it's environment and an App that can display these information.
+* ![oshw] ![board] ![app] [Marian's Business Card](https://github.com/maribu/business-card): Hardware description, board support, games and text display software for business card sized board with USB power supply.
```
Comment (maribu) [inline README.md:34]: This is indeed open hardware and open software.

File: README.md [hunk 2]
```diff
@@ -23,11 +23,21 @@ As part of the RIOT project, the [RIOT code of conduct] applies.
...
+* ![oshw] ![board] [Marian's demo boards](https://github.com/maribu/boards): Various PCBs, including an RGB Christmas tree board, along with RIOT board support for most boards.
```
Comment (maribu) [inline README.md]: For this I have not published a license, making this technically proprietary. I would need to spend some time on this, as I cannot remember if I did create all footprints and symbols from the local lib myself, or if copied some open hardware designs.

I guess the more honest thing to do would be to not claim open hardware here. I created an issue to fix this: https://github.com/maribu/boards/issues/1

**Review by maribu — APPROVED** [review]
Lgtm, please look inline, though.

File: README.md (hunk 2 above)
Comment (chrysn) [reply README.md]: Thanks, OSHW logo removed.

**Comment by chrysn** [comment]
Thanks, adjusted to your comments and merging.

I'd like to do better on the structuring front, but this will be tricky, for we can likely have any combination of properties.

**Pull request merged**